peering/models/models.py

```diff
@@ -541,6 +503,42 @@ def __str__(self):
     def get_absolute_url(self):
         return reverse("peering:directpeeringsession_details", kwargs={"pk": self.pk})
 
+    def merged_export_policies(self):
+        # Get own policies
```

Comment:
I'd probably add a brief description of what this functions purpose is. Same with `merged_import_policies`